src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$size: 400px;
```

Comment:
what exactly is `size`? The name of the variable should describe what it is used for